Index: units/nightmares/ni_hd_01_howling_darkness.cfg
===================================================================
--- units/nightmares/ni_hd_01_howling_darkness.cfg	(revision 14203)
+++ units/nightmares/ni_hd_01_howling_darkness.cfg	(working copy)
@@ -17,7 +17,7 @@
     advances_to="Cloud of Gloom"
     usage="archer"
 
-    description= _ "incomplete"+{SPECIAL_NOTES}+{SPECIAL_NOTES_SKIRMISHER}+{SPECIAL_NOTES_HORROR}
+    description= _ "Meloen became obsessed with the fragility of life and the randomness of fate. Terrible visions beset her: an infant grown breathless and cold in the crib, its sleeping parents an arm's length away, a child fallen in a cistern, unable to swim or call for aid, a viper coiled in the flowerbed. The future, veiled in uncertainty, seemed to hold only the blackest of promises. As Meloen raged at these visions, the Howling Darkness coalesced."+{SPECIAL_NOTES}+{SPECIAL_NOTES_SKIRMISHER}+{SPECIAL_NOTES_HORROR}
 
     die_sound="wail-long.wav"
     image="units/howling_darkness.png"
Index: units/nightmares/ni_lt_01_life_thief.cfg
===================================================================
--- units/nightmares/ni_lt_01_life_thief.cfg	(revision 14203)
+++ units/nightmares/ni_lt_01_life_thief.cfg	(working copy)
@@ -17,9 +17,10 @@
     movement_type="nom_darkness"
     usage="fighter"
 
-    description= _ "In her moments of happiness, Meloen enjoyed gazing at butterflies. In these rare moments of meditation she also got conscious of how much they were short lived and realised her own condition of life.
-During a sleep, while she was pregnant, Meloen saw a spooky spirit shaped like a butterfly, floating above her and the child she was holding, and immediately identified the spirit as an underling of death itself, warning her that she wouldn't live long enough to see her child growth. There was little time then, before spectres of death spread into the forest."+{SPECIAL_NOTES}+{SPECIAL_NOTES_DRAIN}
+    description= _ "In her moments of happiness, Meloen enjoyed gazing at butterflies. In these rare moments of meditation she became conscious of their short lives and felt her own mortality deeply.
 
+While pregnant, Meloen dreamed of an eerie spirit shaped like a butterfly, floating above her and the child she was holding. She immediately identified the spirit as an underling of death itself, warning her that she wouldn't live long enough to see her unborn child grow. There was little time then, before spectres of death spread into the forest."+{SPECIAL_NOTES}+{SPECIAL_NOTES_DRAIN}
+
     die_sound="wail-long.wav"
     image="units/life_thief.png"
     ellipse="misc/ellipse"
Index: units/nightmares/ni_hd_03_eternal_night.cfg
===================================================================
--- units/nightmares/ni_hd_03_eternal_night.cfg	(revision 14203)
+++ units/nightmares/ni_hd_03_eternal_night.cfg	(working copy)
@@ -18,7 +18,8 @@
     usage="archer"
     {AMLA_DEFAULT}
 
-    description= _ "incomplete"+{SPECIAL_NOTES}+{SPECIAL_NOTES_SKIRMISHER}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_HORROR}
+    description= _ "King Dowrein I, Memoirs of a mercenary:
+'Though dawn should have come, the darkness did not lift, nor would it ever for many of my companions. It had decended on us like the shroud of some dreadful reaver, drifting past our unsuspecting pickets and achieving total surprise. Over the baleful howling of our foe could be heard the hopeless screams of its victims. I dropped my useless sword and fled for the river and its doubtful safety.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_SKIRMISHER}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_HORROR}
 
     die_sound="wail-long.wav"
     image="units/eternal_night.png"
Index: units/nightmares/ni_hd_02_cloud_of_gloom.cfg
===================================================================
--- units/nightmares/ni_hd_02_cloud_of_gloom.cfg	(revision 14203)
+++ units/nightmares/ni_hd_02_cloud_of_gloom.cfg	(working copy)
@@ -17,7 +17,8 @@
     advances_to="Eternal Night"
     usage="archer"
 
-    description= _ "incomplete"+{SPECIAL_NOTES}+{SPECIAL_NOTES_SKIRMISHER}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_HORROR}
+    description= _ "King Dowrein I, Memoirs of a mercenary:
+'The moon was long set when we heard the wailing again, a terrible sound that seemed to come from everywhere, promising infinite pain. But this time we spied it racing towards us through the trees... there among the darkest shadows, something blacker yet, like a hole in the night.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_SKIRMISHER}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_HORROR}
 
     die_sound="wail-long.wav"
     image="units/cloud_of_gloom.png"
Index: units/nightmares/ni_lt_02_soul_snatcher.cfg
===================================================================
--- units/nightmares/ni_lt_02_soul_snatcher.cfg	(revision 14203)
+++ units/nightmares/ni_lt_02_soul_snatcher.cfg	(working copy)
@@ -18,9 +18,9 @@
     usage="fighter"
 
     description= _ "King Dowrein I, Memoirs of a mercenary:
-'I had seen this scene a hundred times, the quiet walk of the merchants, the soldiers narrating their exploit of the day before, then the anxiety of the horses, the stop of the march and all eyes and hears looking for the smallest sign. And the attack of those demonic creatures.
+'I have experienced this scene a hundred times, as one merchant caravan or another wound its way along the verge of the deep woods: the quiet conversations of the wagoneers, the soldiers boastfully narrating their exploits of the day before, then the sudden anxiety of the horses, their fearful whinnying and rearing. Wagons and men alike stopped, all eyes and hears searching for the smallest sign. And then, the sudden, brutal attack of those demonic creatures.
 
-But as used as we were to drive them back to that cursed forest, that fear remained the same each with each battle. The fear to lose our mind and soul to one of them, and ultimately, our life.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_DRAIN}
+But though with time and hard-won experience we became skilled at driving them back to that cursed forest, our fear remained the same with each battle: the fear of losing our minds and souls, and ultimately, our lives.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_DRAIN}
 
     die_sound="wail-long.wav"
     image="units/soul_snatcher.png"
Index: units/insects/in_cr_00_creepers.cfg
===================================================================
--- units/insects/in_cr_00_creepers.cfg	(revision 14203)
+++ units/insects/in_cr_00_creepers.cfg	(working copy)
@@ -1,6 +1,13 @@
 #textdomain wesnoth-Nightmares_of_Meloen
 
 # Insects. Creepers, lvl0.
+#define INSECT_COMMON_DESCRIPTION
+_ "Lieutenant Artingale:
+'These degenerated insects band together in large swarms to wreck havoc on living beings in an irrationally aggressive way. An increasing number of reports from the outlying farmsteads describe clouds of bees behaving in this abnormal manner. Some devilry must lie behind it, and I know which unlucky company will be ordered to investigate.'
+
+Note: Isolated creepers can do but little harm, while large bands can pose a threat to any creature."
+#enddef
+
 [unit_type]
     id="Creepers"
     name= _ "Creepers"
Index: units/aberrations/ab_sw_03_inquisitor.cfg
===================================================================
--- units/aberrations/ab_sw_03_inquisitor.cfg	(revision 14203)
+++ units/aberrations/ab_sw_03_inquisitor.cfg	(working copy)
@@ -17,7 +17,8 @@
     movement_type="nom_eye"
     usage="archer"
 
-    description= _ "Bad dreams after bad dreams, growing to nightmares, Meloen fell into a paranoid dementia, thinking to be judged for her misdeeds and revived her childhood, when she was afraid of the way people looked at her. The eyes of disdain were born from this fear and now attack mentally their victims, turning their complexes into physical sufferings."+{SPECIAL_NOTES}+{SPECIAL_NOTES_ARCANE_FOCUS}
+    description= _ "King Dowrein I, Memoirs of a mercenary:
+'I could feel the heat of the creature's gaze, even through my shield. It's disdain for me was like a crushing physical presence. Poor Thornton on my left took the full force of the attack, and though I risked my own life dragging him back to the village, the healers could do nothing to save the wretch.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_ARCANE_FOCUS}
 
     die_sound="ghoul-hit.wav"
     image="units/inquisitor.png"
Index: units/aberrations/ab_sw_02_spiteful_watcher.cfg
===================================================================
--- units/aberrations/ab_sw_02_spiteful_watcher.cfg	(revision 14203)
+++ units/aberrations/ab_sw_02_spiteful_watcher.cfg	(working copy)
@@ -17,7 +17,8 @@
     movement_type="nom_eye"
     usage="archer"
 
-    description= _ "Bad dreams after bad dreams, growing to nightmares, Meloen fell into a paranoid dementia, thinking to be judged for her misdeeds and revived her childhood, when she was afraid of the way people looked at her. The eyes of disdain were born from this fear and now attack mentally their victims, turning their complexes into physical sufferings."+{SPECIAL_NOTES}+{SPECIAL_NOTES_ARCANE_FOCUS}
+    description= _ "Silver Mage Norsesh, Arcane Interactions - Analysis and Theorisation:
+'Intriguingly, the mental focus necessary to achieve substantial arcane effects is not without its liabilities. I hypothesize that the channels we develop in our minds are somehow accessible by certain external forces. The case of the Spiteful Watcher, an arcane creation to be found (at great peril to the finder) in southern Wesnoth, supports this hypothesis. These large, aggressive Watchers posess mental attacks that are especially dangerous for mages.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_ARCANE_FOCUS}
 
     die_sound="ghoul-hit.wav"
     image="units/spiteful_watcher.png"
Index: units/aberrations/ab_un_03b_reckless_one.cfg
===================================================================
--- units/aberrations/ab_un_03b_reckless_one.cfg	(revision 14203)
+++ units/aberrations/ab_un_03b_reckless_one.cfg	(working copy)
@@ -17,7 +17,7 @@
     movement_type="nom_egg"
     usage="fighter"
 
-    description= _ "These hideous monsters came from the very first fear of Meloen: to mother a physically or mentally ill child. Not quite formed yet, the unhatched partly breaks the shell with its powerful legs, then its naturally agressive instinct guide it to charge and complete its hatch onto foes."+{SPECIAL_NOTES}+{SPECIAL_NOTES_CHARGE}
+    description= _ "Dazed, Artingale staggered across the clearing, stepping over the shattered bones and dismembered limbs that littered the ground. The creature had burst suddenly from the undergrowth, giving his men no time to prepare a defense. In a matter of moments a third of the company had been mashed into paste, and the beast had raged off down the trail, evidently unconcerned with the terrible wounds it had taken in return."+{SPECIAL_NOTES}+{SPECIAL_NOTES_CHARGE}
 
     die_sound="yeti-die.ogg"
     image="units/reckless_one.png"
Index: units/aberrations/ab_un_02b_rash_one.cfg
===================================================================
--- units/aberrations/ab_un_02b_rash_one.cfg	(revision 14203)
+++ units/aberrations/ab_un_02b_rash_one.cfg	(working copy)
@@ -17,7 +17,8 @@
     movement_type="nom_egg"
     usage="fighter"
 
-    description= _ "These hideous monsters came from the very first fear of Meloen: to mother a physically or mentally ill child. Not quite formed yet, the unhatched partly breaks the shell with its powerful legs, then its naturally agressive instinct guide it to charge and complete its hatch onto foes."+{SPECIAL_NOTES}+{SPECIAL_NOTES_CHARGE}
+    description= _ "King Dowrein I, Memoirs of a mercenary:
+'They're bad enough still in the shell. Once hatched, the particularly nasty-tempered ones could knock a man clean off the saddle in one blow. On more than one occasion the ugly brutes used their speed to get into our rear, where they did terrible damage to our injured soldiers.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_CHARGE}
 
     die_sound="yeti-die.ogg"
     image="units/rash_one.png"
Index: units/aberrations/ab_un_02_angry_one.cfg
===================================================================
--- units/aberrations/ab_un_02_angry_one.cfg	(revision 14203)
+++ units/aberrations/ab_un_02_angry_one.cfg	(working copy)
@@ -17,7 +17,8 @@
     usage="fighter"
     movement_type="nom_egg"
 
-    description= _ "These hideous monsters came from the very first fear of Meloen: to mother a physically or mentally ill child. Not quite formed yet, the unhatched partly breaks the shell with its powerful legs, then its naturally agressive instinct guide it to charge and complete its hatch onto foes."+{SPECIAL_NOTES}+{SPECIAL_NOTES_BERSERK}
+    description= _ "King Dowrein I, Memoirs of a mercenary:
+'They were bad enough still in the shell. Once hatched, the especially aggressive ones attacked without pause, hurling themselves at us again and again, battling unto death.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_BERSERK}
 
     die_sound="hiss-die.wav"
     image="units/angry_berserk_one.png"
Index: units/aberrations/ab_un_01_unhatched.cfg
===================================================================
--- units/aberrations/ab_un_01_unhatched.cfg	(revision 14203)
+++ units/aberrations/ab_un_01_unhatched.cfg	(working copy)
@@ -17,7 +17,7 @@
     advances_to="Angry One, Rash One"
     usage="fighter"
 
-    description= _ "These hideous monsters came from the very first fear of Meloen: to mother a physically or mentally ill child. Not quite formed yet, the unhatched partly breaks the shell with its powerful legs, then its naturally agressive instinct guide it to charge and complete its hatch onto foes."+{SPECIAL_NOTES}+{SPECIAL_NOTES_CHARGE}
+    description= _ "These hideous monsters came from the very first fear of Meloen: to mother a physically or mentally ill child. Not quite formed yet, the unhatched partly breaks the shell with its powerful legs, then its naturally agressive instinct guides it to complete its hatch by charging into foes."+{SPECIAL_NOTES}+{SPECIAL_NOTES_CHARGE}
 
     die_sound="skeleton-die-1.ogg"
     image="units/unhatched.png"
Index: units/aberrations/ab_un_03_raging_one.cfg
===================================================================
--- units/aberrations/ab_un_03_raging_one.cfg	(revision 14203)
+++ units/aberrations/ab_un_03_raging_one.cfg	(working copy)
@@ -17,7 +17,10 @@
     movement_type="nom_egg"
     usage="fighter"
 
-    description= _ "These hideous monsters came from the very first fear of Meloen: to mother a physically or mentally ill child. Not quite formed yet, the unhatched partly breaks the shell with its powerful legs, then its naturally agressive instinct guide it to charge and complete its hatch onto foes."+{SPECIAL_NOTES}+{SPECIAL_NOTES_BERSERK}
+    description= _ "Excerpt from a conversation between Lieutenant Artingale and Sage Hogene:
+'- Hogene, I often thank the Gods that the biggest Ragers are so very rare.
+- Are they so much more dangerous than the other unnatural denizens of the forest, my Lord?
+- Well, they kill you honestly, which is more than you can say for most. But the bloody smears they leave in their wake haunt my dreams.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_BERSERK}
 
     die_sound="hiss-die.wav"
     image="units/raging_berserk_one.png"
Index: units/elementals/el_ue_02_nature_wrath.cfg
===================================================================
--- units/elementals/el_ue_02_nature_wrath.cfg	(revision 14203)
+++ units/elementals/el_ue_02_nature_wrath.cfg	(working copy)
@@ -28,10 +28,10 @@
     name= _ "Tornado Form of Nature Wrath"
     movement_type="nom_elemental_tornado"
 
-    description= _ "Silver Mage Norsesh, Arcane Control - Analysis and Theorisation:
-'One may interrogate himself on the way to be more potent with the usage of what is called magic by the common. That question inherently comes with an opposition between the quantity of energy transmitted and the efficiency of that transmission.
+    description= _ "Silver Mage Norsesh, Arcane Interactions - Analysis and Theorisation:
+'We now consider the limits of potency in the usage of what is called magic by the common.... Control, in the absence of mental discipline, does not rise commensurately with increasing power. The critical factor appears to be opposition between the quantity of energy transmitted and the efficiency of that transmission.
 [...]
-A literal example of power increase is the growth of Unstable elementals, becoming incarnations of the very wrath of nature, as seen in the south of the Wesnoth region. Far from getting more control over the energy projected, the creature only evolves in increasing that energy, making itself the puppet of its own emotion, like a child with the overgrown body of a man.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_TERRAIN_LINK}
+A literal example of the inefficiency inherent in power increase is the growth of Unstable Elementals, beings found in the south of the Wesnoth region. In their late stage the creatures become incarnations of the very wrath of nature. Far from exerting greater control over their ever increasing energy, the elemental becomes the puppet of its own excess emotion, like a child trapped within the overgrown body of a man, changing from one wild state to another in response to its environment.'"+{SPECIAL_NOTES}+{SPECIAL_NOTES_TERRAIN_LINK}
 
     die_sound="slowed.wav"
     image="units/unstable_tornado_form.png"
